File: pset6/mario.py
import cs50

# Main function
def main():
    # prompt for input
    while True:
        print("Height: ", end="")
        height = cs50.get_int()
        if height >= 0 and height <= 23:
            break
    # return output
    for i in range (height):
        for j in range (height - i - 1):
            print(" ", end="")
        for k in range (height, height - i - 1, -1):
            print ("#", end="")
        print (" ", end="")
        for k in range (height, height - i - 1, -1):
            print ("#", end="")
        print("")

if __name__ == "__main__":
    main()

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        printf("Height: ");
        height = get_int();
    }
    while (height < 0 || height > 23);
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j <= height - i - 2; j++)
        {
            printf(" ");
        }
        for (int k = height; k > height - i - 1; k--)
        {
            printf("#");
        }
        printf("  ");
        for (int k = height; k > height - i - 1; k--)
        {
            printf("#");
        }
        printf("\n");
    }
}
